src/Npgsql/ConnectorPool.cs

```diff
@@ -119,28 +121,31 @@ internal ConnectorPool(NpgsqlConnectionStringBuilder settings, string connString
             if (settings.MaxPoolSize < settings.MinPoolSize)
                 throw new ArgumentException($"Connection can't have MaxPoolSize {settings.MaxPoolSize} under MinPoolSize {settings.MinPoolSize}");
 
-            Settings = settings;
-
-            _max = settings.MaxPoolSize;
-            _min = settings.MinPoolSize;
-
-            UserFacingConnectionString = settings.PersistSecurityInfo
-                ? connString
-                : settings.ToStringWithoutPassword();
-
-            var connectionIdleLifetime = TimeSpan.FromSeconds(Settings.ConnectionIdleLifetime);
-            _pruningSamplingInterval = TimeSpan.FromSeconds(Settings.ConnectionPruningInterval);
-            if (connectionIdleLifetime < _pruningSamplingInterval)
+            if (settings.ConnectionPruningInterval == 0)
+                throw new ArgumentException("ConnectionPruningInterval can't be 0.");
+            var connectionIdleLifetime = TimeSpan.FromSeconds(settings.ConnectionIdleLifetime);
```

Comment:
Let's add an empty line after `throw` like elsewhere.

Reply:
There's no empty line here because I wanted to group the checks